README.md

```diff
@@ -103,6 +103,9 @@
 * [Neural Draw Machine](https://codepen.io/amilajack/pen/xWwONL)
 * [drumbot](https://drumbot.glitch.me)
 
+## NodeJS Addons
+* [`vst.js`](https://github.com/ramirezd42/vst-js)
```

Comment:
Should be added to vst section. Also here's a few suggestions for future PR's:

1. Title shouldn't contain issue ticket number. PR details should.
2. Also consider the following template for future PRs:

pr title: add some-audio-project to some-section
pr details: closes #xxx

This format is simpler (yet contains the same amount of information) and easier to review

Reply:
Thanks for outlining this out. I went ahead and just made a pull request issue template in #98.